### DISCUSSION TITLE
EddieHub domain ... ?

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#790] [Q&A] [answered]
What do you prefer...

- eddiehub.org
- eddiehub.tech
-  eddiehub.io

Please vote below with 👍 👎

**Comment by eddiejaoude** [comment, marked as answer]
eddiehub.org

**Reply by jai-dewani** [reply]
Is it a coincidence that `.org` is the cheapest and everyone voted for it? 😂

**Reply by eddiejaoude** [reply]
lol not sure 🤓  but does that make it **win win** 🎉

**Reply by eddiejaoude** [reply]
This domain is the winner 🎉  now purchased!

**Comment by eddiejaoude** [comment]
eddiehub.tech

**Reply by naomi-lgbt** [reply]
`.tech` not `.tect`?

**Reply by eddiejaoude** [reply]
Oops yes! Thank you 🤓

**Reply by starlightknown** [reply]
Second one!

**Reply by eddiejaoude** [reply]
@kcoder63 can you up vote please

<img width="374" alt="Screenshot 2021-01-27 at 17 59 22" src="https://user-images.githubusercontent.com/624760/106033223-6a1e5d80-60c9-11eb-8f21-b7ba574d768d.png">

**Reply by starlightknown** [reply]
Oh I did the 2nd vote here is mine 😅

**Reply by eddiejaoude** [reply]
oh ok, my bad 🤦‍♂️

**Reply by farhanashrafdev** [reply]
.TECH COZ .ORG IS FOR ORGANIZATIONS

**Reply by naomi-lgbt** [reply]
Please use the upvote feature to vote for your desired choice 🙂

**Reply by farhanashrafdev** [reply]
i did it, but was just giving a remark :D

**Comment by eddiejaoude** [comment]
eddiehub.io

**Comment by adityaraute** [comment]
In my opinion, it shouldn't matter. Pick the cheapest one. It's not like people are gonna get confused. There is only going to be one eddiehub eventually.

My vote goes to the cheapest one, whichever it is.

**Reply by eddiejaoude** [reply]
yeah true 👍 - thank you 🤓

**Reply by adityaraute** [reply]
Well which one is it? 😂

**Reply by eddiejaoude** [reply]
`.org` is the cheapest
